tensorboard/plugins/mesh/mesh_demo.py

```diff
@@ -20,27 +20,38 @@
 import numpy as np
 import tensorflow as tf
 
-from tensorboard.plugins.mesh import summary as mesh_summary
+from tensorboard.plugins.mesh import summary_v2 as mesh_summary
 from tensorboard.plugins.mesh import demo_utils
 
+
 flags.DEFINE_string(
     "logdir", "/tmp/mesh_demo", "Directory to write event logs to."
 )
 flags.DEFINE_string("mesh_path", None, "Path to PLY file to visualize.")
 
 FLAGS = flags.FLAGS
 
-tf.compat.v1.disable_v2_behavior()
-
 # Max number of steps to run training with.
 _MAX_STEPS = 10
 
 
+def train_step(vertices, faces, colors, config_dict, step):
+    """Executes summary as a train step."""
+    # Change colors over time.
+    t = float(step) / _MAX_STEPS
+    transformed_colors = t * (255 - colors) + (1 - t) * colors
+    mesh_summary.mesh(
+        "mesh_color_tensor",
+        vertices=vertices,
+        faces=faces,
+        colors=transformed_colors,
+        config_dict=config_dict,
+        step=step,
+    )
+
+
 def run():
-    """Runs session with a mesh summary."""
-    # Mesh summaries only work on TensorFlow 1.x.
-    if int(tf.__version__.split(".")[0]) > 1:
-        raise ImportError("TensorFlow 1.x is required to run this demo.")
+    """Runs training steps with a mesh summary."""
     # Flag mesh_path is required.
```

Comment:
Optional since this isn't really TF 2.0 related, but it would be nice if this was self-contained. As reported by @wchargin (oh hai) on #2578 there is a .ply file in-repo that we could bundle to make this simpler to run:

https://github.com/tensorflow/tensorboard/blob/master/tensorboard/plugins/mesh/test_data/icosphere.ply

Reply:
Done.